src/components/Common/Search/GlobalSearchBar.tsx

```diff
@@ -62,86 +62,84 @@ const GlobalSearchBar: FC<Props> = ({ onSearchResults }) => {
   }
 
   return (
-    <div className="w-96">
-      <div ref={resultsRef}>
-        <div className="relative mt-1">
-          <div className="relative w-full overflow-hidden border border-gray-200 cursor-default dark:border-gray-800 rounded-xl sm:text-sm">
-            <input
-              className="w-full py-2 pl-3 pr-10 text-sm bg-transparent focus:outline-none"
-              onChange={(event) => setKeyword(event.target.value)}
-              placeholder="Search by hashtag / channel"
-              value={keyword}
+    <div ref={resultsRef}>
```

Comment:
The width is added to have fixed width on the header. Removing it will misalign the positioning.

You can use tailwind breakpoints to just have the width on bigger screens. (`md:w-96`)

Reply:
You're right! Resolved on the last commit!